les/vflux/server/prioritypool.go

```diff
@@ -224,13 +216,13 @@ func (pp *PriorityPool) RequestCapacity(node *enode.Node, targetCap uint64, bias
 	_, minPriority = pp.enforceLimits()
 	// if capacity update is possible now then minPriority == math.MinInt64
 	// if it is not possible at all then minPriority == math.MaxInt64
-	allowed = priority > minPriority
+	allowed = priority >= minPriority
```

Comment:
Is it possible to return the update list from the `enforceLimits`? It's weird to save it in the priority queue.

Reply:
Maybe it could be made nicer, I need to think a bit about it.

Reply:
Now I think that the temporary state does make sense because in some cases I do call `enforceLimits` multiple times and decide the next action based on the temporary state resulting from the previous iteration. It was probably hard to follow though how the temporary state works so I added a commit that does some supposedly identical changes that hopefully makes it cleaner. Now the `capacity` field is not changed until the temp state is committed and there is a `tempCapacity` so it's always clear what we are operating on.